server.go

```diff
@@ -56,22 +76,32 @@ type udpClientTimeoutMap struct {
 	Mux sync.Mutex
 }
 
-var udpPort = 3050
-var tcpPort = 3051
-var version = "0.0.0-development"
-
+const udpPort = 3050
+const tcpPort = 3051
+const atPort = 3060
+const version = "0.0.0-development"
```

Comment:
```suggestion
var udpPort = 3050
var tcpPort = 3051
var atPort = 3060
var version = "0.0.0-development"
```

Must be `var` so it can be replaced during build.